File: BEE 1134.java
import java.io.IOException;
import java.util.Scanner;

public class Main{
    public static void main(String[] args) throws IOException{
        Scanner input = new Scanner(System.in);
        int a = 0;
        int g = 0;
        int d = 0;
        while(true){
            int num = input.nextInt();
            if (num == 4){
                break;
            }
            else if (num == 1){
                a += 1;
            }
            else if (num == 2){
                g += 1;
            }
            else if (num == 3){
                d += 1;
            }
            else{
                continue;
            }
        }
        System.out.println("MUITO OBRIGADO");
        System.out.printf("Alcool: %d\n", a);
        System.out.printf("Gasolina: %d\n", g);
        System.out.printf("Diesel: %d\n", d);
    }
}

File: BEE 1134.py
a = 0
g = 0
d = 0
while(True):
    num = int(input())
    if (num == 4):
        break
    elif (num == 1):
        a += 1
    elif (num == 2):
        g += 1
    elif (num == 3):
        d += 1
    else:
        continue

print("MUITO OBRIGADO")
print("Alcool: %d"% a)
print("Gasolina: %d"% g)
print("Diesel: %d"% d)